pyiceberg/io/pyarrow.py

```diff
@@ -350,7 +351,7 @@ def parse_location(location: str) -> Tuple[str, str, str]:
             return uri.scheme, uri.netloc, f"{uri.netloc}{uri.path}"
 
     def _initialize_fs(self, scheme: str, netloc: Optional[str] = None) -> FileSystem:
-        if scheme in {"s3", "s3a", "s3n"}:
+        if scheme in {"s3", "s3a", "s3n", "oss"}:
```

Comment:
lets also update the docs to mention `oss` 
https://py.iceberg.apache.org/configuration/#fileio

Reply:
should we also add `oss` to fsspec? 
https://github.com/apache/iceberg-python/blob/68e17af746021f630b7efa38a50b3ffb51a2dad8/pyiceberg/io/fsspec.py#L245-L255

Reply:
looks like theres https://github.com/fsspec/ossfs

Reply:
> looks like theres https://github.com/fsspec/ossfs

The project hasn't been maintained for almost a year now, and when I tried it on DuckDB it returns an unknown dependency error (?). I'd say it's probably safer to use s3fs than ossfs (not that I've tried it, though). I'll try making a new pull request for that.

Reply:
makes sense, thanks for looking into it! lets keep the scope just pyarrow for now